app/views/forums/index.html.haml

```diff
@@ -7,7 +7,7 @@
 - @forums.each do |forum|
   %h2= forum
   %p
-    = pluralize(forum.posts.size, "post")
+    = localize_plural(forum.posts, Post)
```

Comment:
Fair enough! I wonder what magic rails is doing under the hood to know what class name to .create or .build. Brief googling did not assist